book/forms.md

```diff
@@ -239,10 +237,10 @@ resolve the block-mixed-with-inline-siblings problem (see
 a node, we assume it's a block element. But `<input>` elements don't have
 children, yet must have inline layout or else they won't draw correctly.
 Likewise, a `<button>` does have children, but they are treated
-specially.^[This situation is specific to these elements in our browser, but
-only because they are the only elements with special painting
-behavior within an inline context. These are also two examples of
-[atomic inlines](https://www.w3.org/TR/CSS2/visuren.html#inline-boxes).]
+specially.^[These are two examples of
+[atomic
+inlines](https://www.w3.org/TR/CSS2/visuren.html#inline-boxes)---the
+only two examples in our browser so far.]
```

Comment:
I think the original was hard to parse.

Reply:
I think the original text is ok?

Reply:
Not critical but I do think it’s an improvement. Perhaps not worth it.

Reply:
The new text also loses information..

Reply:
Reverted.